flashinfer/prefill.py

```diff
@@ -2002,7 +2002,12 @@ def run(
             ]
 
         self._cached_module.paged_run(*run_args)
-
+        if v_scale is not None:
+            # TODO(Zihao): fused into kernel
+            if out.itemsize == 1:
```

Comment:
Is this to detect FP8 and FP4 output? If so, I think we should be more explicit (checking FP8 and UINT8) or at least add a comment.